src/collections/blog/2020-06-25-SMI-conformance-testing-with-meshery/index.mdx

```diff
@@ -46,9 +46,9 @@ Sounds fun, right? Let's dig deeper into the SMI conformance project and find ou
 
 ### Meshery'ing with Conformance Tests
 
-<img src={checklist} align="left" alt="checklist" width="250px" />
+<img src={checklist} align="left" alt="checklist" width="250px" style="margin:1rem 3rem'/>
 
-**[Meshery](/meshery)** is _the service mesh management plane__. It supports all the popular meshes, teaches you how to manage them, assists you in applying custom or recommended configurations, tests for  compatibility, performs performance tests for meshes and a lot more. The SMI conformance testing requires performance testing capabilities, load generation (Meshery is about to support distributed load generation as well), and other functionalities such that these conformance tests can be easily used in the pipelines of all the popular service meshes. mesheryctl does have a perf command that can be used in the pipelines of service meshes. I aim at making such capabilities for SMI conformance as well.
+[Meshery](/meshery) is _the service mesh management plane__. It supports all the popular meshes, teaches you how to manage them, assists you in applying custom or recommended configurations, tests for  compatibility, performs performance tests for meshes and a lot more. The SMI conformance testing requires performance testing capabilities, load generation (Meshery is about to support distributed load generation as well), and other functionalities such that these conformance tests can be easily used in the pipelines of all the popular service meshes. mesheryctl does have a perf command that can be used in the pipelines of service meshes. I aim at making such capabilities for SMI conformance as well.
```

Comment:
To avoid a redirect, will you use "/service-mesh-management/meshery" instead?

Should we import `Link` and use that tag here? @Tanuj22 @Nikhil-Ladha is there any benefit to using the Link component versus an A tag?

Reply:
The  benefit would be that, `Link` will not reload the site. So yes its better to use Link